src/Symfony/Component/Translation/MessageCatalogue.php

```diff
@@ -220,12 +231,14 @@ public function addResource(ResourceInterface $resource)
     /**
      * {@inheritdoc}
      */
-    public function getMetadata($key = '', $domain = 'messages')
+    public function getMetadata($key = '', $domain = null)
```

Comment:
`$key = null` as well then. If we're fixing a consistency issue; let's do it proper. `NULL` means all domains, otherwise (`string`) a given domain (and so for keys).